packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -332,7 +332,7 @@ class LabeledGlobalKey<T extends State<StatefulWidget>> extends GlobalKey<T> {
   ///
   /// The label does not affect the key's identity.
   // ignore: prefer_const_constructors_in_immutables , never use const for this class
-  LabeledGlobalKey(this._debugLabel) : super.constructor();
+  const LabeledGlobalKey(this._debugLabel) : super.constructor();
```

Comment:
This is contradicting the comment right above it.

Reply:
That is a fair point. I was leaning toward more stringent linting but you're right -- we might alarm people unnecessarily.

Reply:
You're right. Removed the const.